include/wx/platform.h

```diff
@@ -611,7 +611,11 @@
         Only 4.3 defines __GXX_RTTI by default so its absence is not an
         indication of disabled RTTI with the previous versions.
      */
-#   if wxCHECK_GCC_VERSION(4, 3)
+#   if defined(__clang__)
+#       if !__has_feature(cxx_rtti)
+#           define wxNO_RTTI
+#       endif
+#   elif wxCHECK_GCC_VERSION(4, 3)
```

Comment:
Clang defines `__GXX_RTTI`, so this could just be
```
#if wxCHECK_GCC_VERSION(4, 3) || defined(__clang__)
```

Reply:
I thought about this, but decided that using clang-specific check was more clear. If you feel otherwise, I can change it, of course, please let me know.